modules/version-update/pages/mtmr-tool.adoc

```diff
@@ -5,13 +5,23 @@
 
 Learn how to use the multi-tenant to multi-runtime tool to convert a multi-tenant platform into a multi-runtime one.
 
-WARNING: Multi-tenancy is deprecated starting from Bonita 2021.1 (technical version 7.12) and will be removed technically with Bonita 2023.1 (technical version 7.16).
+WARNING: xref:ROOT:multi-tenancy-and-tenant-configuration.adoc[Multi-tenancy] is deprecated starting from Bonita 2021.1 (technical version 7.12) and will be removed technically with Bonita 2023.1 (technical version 7.16).
 *If you are using multi-tenancy, you will have to run this conversion tool before updating to 7.16.*
 
 image:version-update:mtmr-general-idea-schema.svg[General idea of the conversion tool,width=700]
 
 == Overview
-The tool will delete everything not related to the tenant specified in the parameter from the given database, this means you will need to dump your current database and mount one new database dump per tenant in your environment. The tool will have to be executed for every database dump.
+
+
+Following the commercial depreciation of multi-tenancy use by customer willing to have separate BDMs, the technical depreciation will be effective with Bonita 2023.1. The technical solution for customers willing to have separate BDMs per applications/processes proposed by Bonitasoft will be multi-runtimes. 
```

Comment:
"for customers willing to have separate BDMs per applications/processes" --> I am not certain that is correct.
If it means BDM as in the BDM database, it's just wrong/weird.
If it means something else in this context, we need to use another term or explicit the abbreviation, because BDM in the documentation always means the BDM database.

Reply:
Changed it with Bonita engine database. Adrien's suggestion :)